docs/devel/pull-requests.md

```diff
@@ -73,12 +74,30 @@ The following will save time for both you and your reviewer:
 
 ## Release Notes
 
-1. Your PR title is the **release note** you want published at release time.
-1. Release note labels are only needed on master branch PRs.
+This section applies only to pull requests on the master branch.
+
 1. All pull requests are initiated with a `release-note-label-needed` label.
 1. For a PR to be ready to merge, the `release-note-label-needed` label must be removed and one of the other `release-note-*` labels must be added.
 1. `release-note-none` is a valid option if the PR does not need to be mentioned
  at release time.
+1. The PR title is the **release note** you want published at release time.
+  * NOTE: PR titles are mutable and should reflect a release note friendly
+    message for any `release-note-*` labeled PRs.
+
+### Reviewing pre-release notes
+
+**COMING SOON**
+
+At any time, you can see what the release notes will look like on any branch.
+
+```
+$ git pull https://github.com/kubernetes/release
```

Comment:
Qualified with **COMING SOON**.  I'm attempting to optimize the fact that every little change to a doc seems to take days or longer due to long test/queue cycles coupled with flakes and retries... *sigh*.